.github/workflows/report_quiz/README.md

```diff
@@ -0,0 +1,69 @@
+REPORT QUIZ BOT
```

Comment:
Good habit, to have a dedicated readme for each our automations

Reply:
There is a log of my tests here (GitHub action).
https://github.com/pythonbrad/global-github-actions/actions

Reply:
Good, add it to the pr description